features/_your_results/period_percent_table.feature

```diff
@@ -0,0 +1,9 @@
+Feature:
+  As a user who has just completed the WPCC tool, I should be able to see helpful information on how the yearly increments is done. 
+
+  Scenario: Display table of percentages for each period
+    Given I am on the YourDetailsPage
+    When I fill in my details
+    And I click the Next button
+    And I move on to the results page
+    Then I should see a table with a column heading for each period
```

Comment:
Is this the only scenario to test it? What happens if the user is contributing more that the period percent? (I am just asking because I can't find the answer on the PR description nor the user story - cc @UmairIft )